charts/loki-distributed/templates/compactor/deployment-compactor.yaml

```
@@ -39,6 +39,9 @@ spec:
         - name: loki
           image: {{ include "loki.compactorImage" . }}
           imagePullPolicy: {{ .Values.loki.image.pullPolicy }}
+          {{- if or .Values.loki.command .Values.compactor.command }}
+          command: [{{ coalesce .Values.compactor.command .Values.loki.command | quote }}]
+          {{- end }}
```

Please use block style for YAML lists. Apply that to all your changes.

```suggestion
          {{- if or .Values.loki.command .Values.compactor.command }}
          command: 
            - {{ coalesce .Values.compactor.command .Values.loki.command | quote }}
          {{- end }}
```